### PR TITLE
Add several `Scaler Topics` tutorials

### PR DESCRIPTION
**Pull request by Vaishnaviamirayada**
Have added a useful resource about Bootstrap from the Scaler topics in the Bootstrap section.

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems. Also check your spelling.

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -180,6 +180,7 @@
 * [Bootstrap 5 Tutorial](https://www.w3schools.com/bootstrap5/) - W3Schools
 * [Bootstrap Tutorial](https://www.tutlane.com/tutorial/bootstrap) - tutlane
 * [Front End Development Libraries Certification: Bootstrap](https://www.freecodecamp.org/learn/front-end-libraries/bootstrap) - freeCodeCamp
+* [Bootstrap Tutorial](https://www.scaler.com/topics/css/bootstrap/) - Scaler Topics
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:183]: Alphabetize to solve lint faults

```suggestion
* [Bootstrap Tutorial](https://www.scaler.com/topics/css/bootstrap/) - Scaler Topics
* [Front End Development Libraries Certification: Bootstrap](https://www.freecodecamp.org/learn/front-end-libraries/bootstrap) - freeCodeCamp
```

**Comment by Vaishnaviamirayada** [comment]
Hi @davorpa,

I have made all the changes as per the guidelines, please check.

Thank You!

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: more/free-programming-interactive-tutorials-en.md [hunk 2]
```diff
@@ -256,6 +261,7 @@
 #### Operating systems
 
 * [Learning operating system development using Linux kernel and Raspberry Pi](https://github.com/s-matyukevich/raspberry-pi-os) - Sergey Matyukevich (:construction: *in process*)
+* [Operating System Tutorial] (https://www.scaler.com/topics/operating-system/) - Scaler Topics
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:264]: No spaces between link markup tokens

```suggestion
* [Operating System Tutorial](https://www.scaler.com/topics/operating-system/) - Scaler Topics
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Review by davorpa — COMMENTED** [review]
@eshellman could you recheck again?

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**